meta-lmp-base/files/lmp-passwd-table

```diff
@@ -17,6 +17,7 @@ irc:x:39:39:ircd:/var/run/ircd:/bin/sh
 gnats:x:41:41:Gnats Bug-Reporting System (admin):/var/lib/gnats:/bin/sh
 nm-openvpn:x:975:975::/home/nm-openvpn:/bin/sh
 mosquitto:x:976:976::/home/mosquitto:/bin/false
+weston:x:63:63::/home/weston:/bin/false
```

Comment:
Can you just reorder and have it bellow gnats? Just to keep the numbering ordered correctly.

Reply:
Done!